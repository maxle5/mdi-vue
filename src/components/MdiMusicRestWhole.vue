
  <template>
      <svg :width="size" :height="size" :viewBox="viewBox">
          <path d="M18 10H16V14H8V10H6V9H18V10Z" style="fill: currentColor" />
      </svg>
  </template>

  <script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiMusicRestWhole",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
  </script>
