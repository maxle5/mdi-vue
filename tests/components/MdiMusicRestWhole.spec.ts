
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiMusicRestWhole from "../../src/components/MdiMusicRestWhole.vue";

test("MdiMusicRestWhole snapshot", () => {
  const wrapper = mount(MdiMusicRestWhole, {});
  expect(wrapper.html()).toMatchSnapshot();
});
