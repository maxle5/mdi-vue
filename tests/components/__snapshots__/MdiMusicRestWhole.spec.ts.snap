// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiMusicRestWhole snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M18 10H16V14H8V10H6V9H18V10Z" style="fill: currentColor;"></path>
</svg>"
`;
